packages/layout/src/Legend.ts

```diff
@@ -251,7 +251,11 @@ export class Legend extends SVGWidget {
         this.updateDisabled(element, dataArr);
 
         const bbox = this.getBBox(true, true);
-        this._g.attr("transform", `translate(${this.width() / 2 - bbox.width / 2 + 5},${this.height() / 2 - bbox.height / 2})`);
+        this._g.attr("transform", "translate(5,8)");
+        this.pos({
+            x: this.width() / 2 - bbox.width / 2,
+            y: this.height() / 2 - bbox.height / 2
+        });
```

Comment:
This new scroll bar feature works on the Legend with just this code change? I would have expected that some custom logic would be needed to read the y-offset of the last ".cell" because the bbox would only reflect the dimensions of the outer svg element?

Reply:
The getBBox reflects this size of the inner items and is used to manually center them (as in this case) - but rather than doing the centering inside the widget, if you let the SVG element do it for you, then it also has the scope to handle the scrolling also.